style.css

```diff
@@ -226,6 +226,75 @@ body {
   color: #fff;
 }
 
+/* mobile popup section */
+.mobile-popup {
```

Comment:
[OPTIONAL] Great job fixing the button design issue for desktop but let's make our modal fully responsive as well.  Please check the button overlapping issue ↓
<img width="858" alt="Screenshot 2023-02-02 at 9 10 07 PM" src="https://user-images.githubusercontent.com/25887253/216413073-d8b414ef-1fc8-499d-adff-b00ab0e8e595.png">